cfme/tests/test_appliance.py

```diff
@@ -408,3 +410,53 @@ def test_appliance_log_error():
         initialEstimate: 1/2h
     """
     pass
+
+
+@pytest.mark.ignore_stream('5.9')
+def test_codename_in_log(appliance):
+    """
+    check whether logs contains a mention of appliance codename
+
+    Polarion:
+        assignee: jhenner
+        casecomponent: Appliance
+        initialEstimate: 1/60h
+    """
+    log = '/var/www/miq/vmdb/log/evm.log'
+    lv = LogValidator(log,
+                      matched_patterns=[r'.*Codename: \w+$'],
+                      hostname=appliance.hostname,
+                      username=conf.credentials['ssh']['username'],
```

Comment:
I think its best to actually not pass hostname, username, or password to LogValidator. It will end up defaulting to SSHClient.__init__'s behavior of using current_appliance, and looking up these credentials.

Reply:
Note I just opened #8645 to make the rest of the uses of LogValidator in the tests more consistent.